src/peergos/server/storage/KeyFilter.java

```diff
@@ -0,0 +1,8 @@
+package peergos.server.storage;
+
+import peergos.shared.crypto.hash.*;
+
+public interface KeyFilter {
```

Comment:
This could be replaced with ``Predicate<PublicKeyHash>>`` which could be clearer.

Reply:
done